src/tools/distillery.ml

```diff
@@ -1,12 +1,28 @@
 open Printf
 
-(* Constant use for the filename containing the reserve project name for
- * templates
- *)
+(* File containing the reserve project names for templates *)
 let reserve_project_name_filename = ".reserve"
 
+(* File containing files which must be ignored while copying a template. *)
+let eliomignore_filename          = ".eliomignore"
```

Comment:
I chose `.eliomignore` because I don't know if some projects use `.ignore`. I mean, for git is `.gitignore` and I suppose there are also other projects which use ignore files and I think is not a good idea to impose `.ignore` for Eliom.

Or maybe replace `.reserve` by `.eliomreserve`.